tools/ocamlmklib.ml

```diff
@@ -63,15 +63,27 @@ let print_version_num () =
   exit 0;
 ;;
 
+let push_args first arr stk =
+  for i = Array.length arr - 1 downto first do
+    Stack.push arr.(i) stk
+  done
+;;
```

Comment:
I would move this within `parse_arguments` to be next to `next_arg`, and use a label for the first argument for readability of the callsites.

Reply:
Yes, done.